Lib/idlelib/configdialog.py

```diff
@@ -1853,6 +1853,219 @@ def save_all_changed_extensions(self):
             self.ext_userCfg.Save()
 
 
+class FontPage:
+
+    def __init__(self, parent, tab_pages, highlight_sample):
```

Comment:
Is there anything that would allow us to create highlight_sample outside of both FontPage and ThemePage and let both of them access it?  A superclass would seem a bit much, but I don't know if can just be in ConfigDialog and then have self.highlight_sample sent to the 2 classes?